RoaringBitmap/src/main/java/org/roaringbitmap/art/Node48.java

```diff
@@ -0,0 +1,168 @@
+package org.roaringbitmap.art;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.Arrays;
+
+public class Node48 extends Node {
+
+  //the value of childIndex content won't be beyond 48
+  byte[] childIndex = new byte[256];
```

Comment:
Since this is used for search operations where we skip over empty values, if this were `new long[32]` we could skip many bytes with a single instruction via `Long.numberOfTrailingZeros`